aspnetcore/web-api/index.md

```diff
@@ -219,6 +219,12 @@ The `ValidationProblemDetails` type:
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-2.1"
```

Comment:
I think we introduced the helpers in 3.0. The type exists in 2.1 though

Reply:
Method https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.validationproblem?view=aspnetcore-2.1#Microsoft_AspNetCore_Mvc_ControllerBase_ValidationProblem exists in 2.1. I was surprised as well.

![image](https://user-images.githubusercontent.com/6978458/87963083-da1bad80-cac0-11ea-983a-85c0a3fecb32.png)

Reply:
The whole topic is set up as 2.1 or later, so the moniker range isn't needed.

Reply:
You're right @sguryev 👍

Reply:
@guardrex OK thank you. I have removed all 2.1 monikers

Reply:
Sure thing. We'll need API xref cross-links and perhaps a *nit* touch to wording. I pinged Scott to take a look. This is a topic that he authored, so it's best if he takes a look.

@scottaddie ... if ur bogged down ... :ear: I can take over with the final touches if you want.

Reply:
@guardrex I'll take care of it right now.